src/coreclr/tests/src/Interop/StringMarshalling/AnsiBSTR/CMakeLists.txt

```diff
@@ -1,17 +1,17 @@
 project (AnsiBStrTestNative)
-include ("${CLR_INTEROP_TEST_ROOT}/Interop.cmake") 
+include ("${CLR_INTEROP_TEST_ROOT}/Interop.cmake")
 set(SOURCES AnsiBStrTestNative.cpp)
 
 # add the executable
 add_library (AnsiBStrTestNative SHARED ${SOURCES})
 
-if(WIN32)
+if(CLR_CMAKE_TARGET_WIN32)
     list(APPEND LINK_LIBRARIES_ADDITIONAL
         OleAut32.lib
```

Comment:
Host specific libraries... Ceasing commenting on this in later files....

Reply:
Changed to HOST in the places you pointed out plus an additional file. Thanks!